meta-aos-vm-common/recipes-aos/aos-clearhsm/aos-clearhsm.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Aos script to clear PKCS11 storage"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://aos-clearhsm.sh \
+"
+
+S = "${WORKDIR}"
+
+FILES:${PN} = " \
+    ${bindir} \
+"
+
+RDEPENDS:${PN} = "aos-setupdisk"
```

Comment:
why is this dependency needed?

Reply:
it is not needed. forgot to remove while adapting aos-deprov.sh